app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -99,4 +102,30 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
     }
     return GithubBuildStatusUpdate.statusSuccess;
   }
+
+  Future<void> _insertBigquery(String buildStatus) async {
+    // Define const variables for [BigQuery] operations.
+    const String projectId = 'flutter-dashboard';
+    const String dataset = 'cocoon';
+    const String table = 'EngineBuildStatus';
```

Comment:
Any reason to have a separate table for engine? can we just use the same one as sdk?

Reply:
If using existing table for sdk, we need to add a new column to distinguish sdk and engine. However Bigquery does not support adding new column on the fly. That means we need to recreate a new table, and then import existing sdk data to the new table.

Reply:
I'm assuming we'll need to do it sooner or later when adding a new repo. The question is do we do it now where we have a single table or we move the effort to merge two tables later?

Reply:
Now we are using the same table to both sdk and engine, and it is easy to extend to other repos as well.